core/src/test/java/org/apache/iceberg/view/ViewCatalogTests.java

```diff
@@ -107,6 +107,7 @@ public void basicCreateView() {
     assertThat(view.currentVersion().operation()).isEqualTo("create");
     assertThat(view.schemas()).hasSize(1).containsKey(0);
     assertThat(view.versions()).hasSize(1).containsExactly(view.currentVersion());
+    assertThat(view.properties()).containsEntry("key1", "catalog-default-key1");
```

Comment:
Minor nit in these tests, I don't think it's clear from looking at this test where the value is coming from. I would at least add a comment 
// Validate Catalog View Default

Another thing we could do is just make a full test that just checks that catalog defaults are set